docs/src/administration/users.md

```diff
@@ -3,136 +3,184 @@ title: "User administration"
+1. In the console, select the project where you want to delete a user.
+2. Under **Settings**, click **Access**.
+3. Select the user you want to delete and click **Delete**.
+4. Click **Save**.
 
-In this example, using either `platform get` with the CLI:
+<--->
 
-```bash
-$ platform get <projectID>
+---
+title=Using the CLI
+file=none
+highlight=false
+---
 
-```
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to delete existing users.
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+  ```bash
+  platform user:delete user1@example.com
+  ```
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
+{{< /codetabs >}}
 
-both would error with
+Once you delete a user, they can no longer access the project.
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment.
+You can redeploy by clicking **Redeploy** in the console or  using the CLI command `platform redeploy`.
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
+### Change existing permissions for environment types
 
-Please make sure you have the correct access rights and the repository exists.
-```
+To change user permissions, follow these steps:
+
+{{< codetabs >}}
+
+---
+title=In the console
+file=none
+highlight=false
+---
 
-despite their **Project Admin** access to the project.
+1. In the console, select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+<--->
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
 ```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
+platform user:update user1@example.com -r production:v,development:c
```

Comment:
What about an introduction? As I said before:
>I think the command isn't immediately obvious, so like the example with adding a user via the CLI, it would help to have an introductory sentence explaining what you're trying to do.

Something like this:, "
```suggestion
Say you want `user1@example.com` to have the Viewer role for Production environments
and the Contributor role for Development environments:

```bash
platform user:update user1@example.com -r production:v,development:c
```
(I can't get the exact lines to match because of the deleted one.)